README.md

```diff
@@ -1,8 +1,8 @@
-# EigenDA Proxy
+# EigenDA Proxy <!-- omit from toc -->
 
-A basic REST server for Rollups with the following endpoints:
-- POST routes: used by sequencers to encode their payloads (batches of transactions or state transitions) into EigenDA blobs, submit them to the EigenDA disperser to make them available, and receive a DA certificate to be submitted to the rollup's batcher inbox.
-- GET routes: used by validators to retrieve the blob from the EigenDA network given a DA Certificate, decode the blob back into the original payload, and return it to the validator, to be used in its derivation pipeline.
+A basic REST proxy server to interact with the EigenDA network:
+- POST routes: submit payloads (rollup txs, state-diffs, or anything really) that will be encoded into EigenDA blobs and submitted to the EigenDA disperser to make available for 2 weeks. A DA certificate of availability will be returned, which can be used to validate the availability and query the payload back.
+- GET routes: submit back the received DA Certificate to retrieve the blob from the EigenDA network, which will be decoded, validated, and returned as a response.
```

Comment:
"submit back the received DA Certificate"

This phrase is awkward, consider simplifying to "submit the DA Certificate"